src/interchange/src/avro.rs

```diff
@@ -896,16 +896,59 @@ fn build_schema(columns: &[(ColumnName, ColumnType)]) -> Schema {
             {
                 "name": "after",
                 "type": ["null", "row"]
+            },
+            {
+                "name": "transaction",
```

Comment:
this purposefully omits the total order / data collection order fields (see: https://debezium.io/documentation/reference/connectors/postgresql.html#postgresql-transaction-metadata ) -- its not impossible to have those but it would require a larger restructuring of the sink code + I am not sure if those orderings are semantically meaningful

I don't know how people feel re: omitting the fields vs including them in the schema and always sending 0?

Reply:
I think it's fine to omit it, but we might want to add a comment to that effect in code? The only place where I can see it being handy is for duplicate messages.